### PR TITLE
v0.6.1 release branch

### PR DESCRIPTION
**Pull request by wsavran**

### CODE REVIEW

File: csep/utils/plots.py [hunk 1]
```diff
@@ -1883,3 +1885,249 @@ def add_labels_for_publication(figure, style='bssa', labelsize=16):
...
+                    height=(yTickPos[1] - yTickPos[0]), color=['w', 'gray'], alpha=0.2, zorder=0)
+    ax.set_xlabel(xlabel, fontsize=14)
+    ax.tick_params(axis='x', labelsize=13)
+    if tight_layout:
+        ax.figure.tight_layout()
+        fig.tight_layout()
+    return ax
+    
+
+def plot_pvalues_and_intervals(test_results, ax, var=None):
```
Comment (wsavran) [inline csep/utils/plots.py:2019]: @bayonato89 i had to make some modifications here to work within pycsep. can you test this new function out and make sure that it's working properly?

Comment (bayonato89) [reply csep/utils/plots.py:2019]: It does work properly @wsavran

**Comment by wsavran** [comment]
@bayonato89, I'm writing some docs and reviewing the `binomial_tests.py`, and i have a couple of questions. 

1. It seems we are missing the function `_simulate_catalog()`. is this from `poisson_evaluations.py`? if not, can you add the binomial `_simulate_catalog()` to this pull request?
2. This also makes me think we should have a test case for this function. Can you give me a worked example that we could implement like [this](https://github.com/SCECcode/pycsep/blob/bb299b2cfc23bc5e2f61d7a91e67d316fad12bff/tests/test_csep1_evaluations.py#L85), for one of the binomial tests with simulation?

**Comment by bayonato89** [comment]
Just added the `_simulate_catalog` function (although I think I sent it to the wrong branch). Let me know if I should resubmit @wsavran. The function was written by @khawajasim. We should be careful when calling it, because the `_simulate_catalog` functions for Poisson and non Poisson consistency tests share the same name.

**Pull request merged**